SPECS/unbound/unbound.spec

```diff
@@ -40,8 +41,7 @@ Group:      Documentation
 unbound dns server docs
 
 %prep
-%setup -q -n %{name}-release-%{version}
-%patch0 -p1
+%autosetup -p1 -n %{name}-release-%{version}
```

Comment:
The `%autopatch` macro used to have issues with the "nopatches". Have you verified the package still builds fine with this change? I see the nopatch file is empty, so I guess it might be fine, but my approval assumes you've doublec-checked that.;)

Reply:
Yes, local build worked on x86_64